docs/developer_setup.md

```diff
@@ -97,6 +97,17 @@ sudo pip3 install pip3 install speechRecognition==3.8.1 service_identity pockets
 sudo pip3 install -r susi_linux/requirements-special.txt
 ```
 
+download speech sata for flite TTS
```

Comment:
Please capitalize the first letter.

Reply:
Done now, also changed other section headers so that the first letter is uppercase